src/Schemes/CoveredSchemes/Objects/Properties.jl

```diff
@@ -20,12 +20,13 @@ end
 end
 
 @attr function is_integral(X::AbsCoveredScheme)
-  return all(U->(is_integral(U)), affine_charts(X))
-  # is_connected is not implemented yet for graphs, so we skip it for the time being
   return is_connected(X) && all(U->(is_integral(U)), affine_charts(X))
 end
 
 @attr function is_connected(X::AbsCoveredScheme)
   return is_connected(glueing_graph(default_covering(X)))
 end
 
+function is_connected(g::Graph{Undirected})
+  return Polymake.call_function(:graph, :is_connected, g.pm_graph)
+end
```

Comment:
@benlorenz : Is it OK to do it like this?

Reply:
```suggestion
is_connected(g::Graph{Undirected}) = Polymake.call_function(:graph, :is_connected, pm_object(g))::Bool
```
Some small improvements: type annotation for better type stability (since julia cannot deduce the return type of `call_function`), shorthand notation, and using `pm_object` is preferred since it makes it easier to refactor if necessary.

Reply:
Alright. Thanks for taking a look!